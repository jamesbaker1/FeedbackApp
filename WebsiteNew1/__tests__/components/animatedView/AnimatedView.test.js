'use strict';

import React          from 'react';
import AnimatedView   from '../../../src/app/components/animatedView/AnimatedView';
import renderer       from 'react-test-renderer';

describe('AnimatedView component', () => {
  it('renders as expected', () => {
    const component = renderer.create(
      <div>
        <AnimatedView>
          <p>
            An animated view content
          </p>
        </AnimatedView>
      </div>
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
