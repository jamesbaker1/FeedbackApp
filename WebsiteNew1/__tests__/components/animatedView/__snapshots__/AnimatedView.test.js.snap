// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnimatedView component renders as expected 1`] = `
<div>
  <section
    className="content invisible"
  >
    <p>
      An animated view content
    </p>
  </section>
</div>
`;
